mne/tests/test_bem.py

```diff
@@ -405,4 +405,40 @@ def test_make_flash_bem(tmpdir):
     plt.close('all')
 
 
+@requires_freesurfer('mri_watershed')
+@requires_nibabel()
+@testing.requires_testing_data
+def test_make_watershed_bem(tmpdir):
```

Comment:
Ahh because this is in `tests/test_bem.py`. Let's just add it to the existing `test_watershed_bem` test in `mne/commands/tests/test_commands.py`. It's properly decorated to deal with the fact that these tests take forever

Reply:
My bad, I would add the additional tests to `test_command.py`. Also there's `test_flash_bem` in both `mne/commands/tests/test_commands.py` and `tests/test_bem.py`, should I move `test_flash_bem` in `mne/tests/test_bem.py` to `mne/commands/tests/test_commands.py` for consistency? `test_flash_bem` in `tests/test_bem.py` looks redundant.